pkg/registry/shadow/template/rest.go

```diff
@@ -160,6 +255,47 @@ func (r *REST) Get(ctx context.Context, name string, options *metav1.GetOptions)
 	return transformManifest(manifest)
 }
 
+// GetFromCCls retrieves the item from child cls.
+func (r *REST) GetFromCCls(ctx context.Context, name string, options *metav1.GetOptions) (runtime.Object, error) {
+	mcls, err := r.getMCLs()
+	if err != nil {
+		return nil, apierrors.NewServiceUnavailable(err.Error())
+	}
+	resource, _ := r.getResourceName()
+	return r.getResourceFromCCls(ctx, strings.ToLower(resource), name, options, mcls)
+}
+
+// getResourceFromCCls get resource From child cluster
+func (r *REST) getResourceFromCCls(ctx context.Context, resource string, name string, options *metav1.GetOptions,
+	mcls []*v1beta1.ManagedCluster) (runtime.Object, error) {
+	var unstructObj *unstructured.Unstructured
+	var err error
+	for _, cls := range mcls {
+		clusterID := string(cls.Spec.ClusterID)
+		clientset, ok := r.getCClsCliFromCache(clusterID)
```

Comment:
A big shout out to how the RABC rules should be set for this up-to-bottom `GET` operations to child clusters.
It seems `admin` account from child cluster is used here for listing operations.